runner/Cargo.toml

```diff
@@ -30,6 +30,7 @@ serde_ipld_dagcbor = "0.6"
 serde_ipld_dagjson = "0.2"
 schemars.workspace = true
 serde_json.workspace = true
+once_cell = "1.19.0"
```

Comment:
nice, just an fyi you can use a [OnceLock](https://doc.rust-lang.org/std/sync/struct.OnceLock.html) from the std library now, but I prefer the API on once_cell tbh